scalalib/overrides/scala/Enumeration.scala

```diff
@@ -214,9 +214,9 @@ abstract class Enumeration (initial: Int) extends Serializable {
       else s"<Unknown name for enum field #$i of class ${getClass}>"
 
     protected def readResolve(): AnyRef = {
-      val enum = thisenum.readResolve().asInstanceOf[Enumeration]
-      if (enum.vmap == null) this
-      else enum.vmap(i)
+      val enumeration = thisenum.readResolve().asInstanceOf[Enumeration]
+      if (enumeration.vmap == null) this
+      else enumeration.vmap(i)
```

Comment:
Don't change files under `scalalib/overrides...`. Those files will never be compiled by Scala 3, and should stay in sync with the corresponding files upstream.

Reply:
The file has been reverted back to `0.6.x`